README.md

```diff
@@ -2,37 +2,56 @@
 
 [![Build Status](https://github.com/pi-hole/docker-pi-hole/workflows/Test%20&%20Build/badge.svg)](https://github.com/pi-hole/docker-pi-hole/actions?query=workflow%3A%22Test+%26+Build%22) [![Docker Stars](https://img.shields.io/docker/stars/pihole/pihole.svg?maxAge=604800)](https://store.docker.com/community/images/pihole/pihole) [![Docker Pulls](https://img.shields.io/docker/pulls/pihole/pihole.svg?maxAge=604800)](https://store.docker.com/community/images/pihole/pihole)
 
-<p align="center">
-<a href="https://pi-hole.net"><img src="https://pi-hole.github.io/graphics/Vortex/Vortex_with_text.png" width="150" height="255" alt="Pi-hole"></a><br/>
-</p>
+<div align="center">
+  <a href="https://pi-hole.net/">
+    <img src="https://pi-hole.github.io/graphics/Vortex/vortex_with_text.svg" width="144" height="256" alt="Pi-hole website">
+  </a>
+  <br>
+  <strong>Network-wide ad blocking via your own Linux hardware</strong>
+  <br>
+  <br>
+  <div align="center">
+    <a href="https://pi-hole.net/">Pi-hole website</a> |
+    <a href="https://docs.pi-hole.net/">Documentation</a> |
+    <a href="https://discourse.pi-hole.net/">Discourse Forum</a> |
+    <a href="https://pi-hole.net/donate">Donate</a>
+  </div>
+  <br>
+  <br>
+</div>
 <!-- Delete above HTML and insert markdown for dockerhub : ![Pi-hole](https://pi-hole.github.io/graphics/Vortex/Vortex_with_text.png) -->
 
 ## Upgrade Notes
 
-## !!! THIS VERSION CONTAINS BREAKING CHANGES
-
-### v[ChangeMeBeforeTagging] has been entirely redesigned from the ground up and contains many breaking changes. Environment variable names have changed, script locations may have changed. Please read the the Readme carefully before proceeding
+> [!CAUTION]
+>
+> ## !!! THIS VERSION CONTAINS BREAKING CHANGES
+> **v6 has been entirely redesigned from the ground up and contains many breaking changes.**
```

Comment:
```suggestion
> **v[ChangeMeBeforeTagging] has been entirely redesigned from the ground up and contains many breaking changes.**
```

I suppose `v[ChangeMeBeforeTagging]` is there as Docker is more concerned about tags in the format `YYYY.MM.X`. The tag cannot be foreseen as there is no fixed `v6` release date.

Reply:
I agree we need a better text here, but the current paragraph is not wrong (maybe just incomplete).

`v6` is an entirely redesigned Pi-hole. Every v6 tag will contain breaking changes compared to v5 tags.

Reply:
I changed the text a little bit to warn that replacing older images (`2024.07.0` and earlier) with newer images will cause irreversible changes to config files in the volumes.

I know that docker is related to image tags, but our image tags doesn't contain Pi-hole version number. It is necessary to reinforce that new images will use Pi-hole v6 to avoid confusion.

Reply:
@DL6ER 

After the recent commits, do you still has any suggestions?